scripts/test-check-packages.sh

```diff
@@ -10,6 +10,13 @@ cleanup() {
 
 trap cleanup EXIT
 
+for d in test/packages/*/; do
+  (
+    cd $d
+    elastic-package build -v
```

Comment:
Could you please double-check if the `check` command doesn't include `build` step?